advertising: Use collection cpm in blurb text.

madbook opened the pull request:
Previously, cpm_selfserve was the lowest cpm, but that changed when we introduced collections. This makes the the text in the 'platform' blurb pull from cpm_selfserve_collection instead.

:eyeglasses: @bsimpson63 @dwick

Commit:
advertising: Use collection cpm in blurb text.

Previously, cpm_selfserve was the lowest cpm, but that changed when we
introduced collections. This makes the the text in the 'platform' blurb
pull from cpm_selfserve_collection instead.

bsimpson63 commented on reddit_about/pages.py:
```diff
@@ -98,7 +98,7 @@ def __init__(self, *args, **kwargs):
         blurbs = SelfServeBlurb.get_all(return_dict=True)
         if 'platform' in blurbs:
-            formatted_cpm = format_currency(g.cpm_selfserve.decimal, 'USD', locale=c.locale)
+            formatted_cpm = format_currency(g.cpm_selfserve_collection.decimal, 'USD', locale=c.locale)
```
Could we do something like `min([g.cpm_selfserve_collection, g.cpm_selfserve, g. cpm_selfserve_geotarget_metro]).decimal`

bsimpson63 commented:
:haircut:

bsimpson63 commented:
:nail_care:

Commit:
fixup! advertising: Use collection cpm in blurb text.

Instead of just pointing to the new lowest cpm value, calculate the min of all
three and use that, so we don't have to update this again (unless we add a
new cpm type into the mix.

madbook commented:
good suggestion! :haircut:

bsimpson63 commented:
:fish:

madbook commented:
3f2c7be7ebbc488c2b4051d7af74c7b5033e940c
